dist/home.css

```diff
@@ -0,0 +1,690 @@
+  .header-nav-one {
+    background-color: #272930;
+  }
+}
+.header-nav-one .nav-one-links {
+  display: inline-flex;
+  align-items: center;
+  margin-top: 0.75rem;
+}
+.header-nav-one .nav-one-links li:nth-of-type(1),
+.header-nav-one .nav-one-links li:nth-of-type(2) {
+  margin-right: 0.75rem;
+}
+@media (min-width: 48em) {
+  .header-nav-one .nav-one-links {
+    margin-top: 0;
+  }
+  .header-nav-one .nav-one-links li:not(:last-child) {
+    margin-right: 0.75rem;
+  }
+}
+.header-nav-two {
+  order: 2;
+}
+@media (min-width: 48em) {
+  .header-nav-two {
+    display: flex;
+    background-color: #fff;
+    padding: 0.75rem 8%;
+    justify-content: space-around;
+    align-items: center;
+  }
+}
+.header-nav-two .logo {
+  display: none;
+}
+@media (min-width: 48em) {
+  .header-nav-two .logo {
+    display: block;
+    width: 13.75rem;
+  }
+}
+.header-nav-two .nav-list {
+  text-align: center;
+}
+@media (min-width: 48em) {
+  .header-nav-two .nav-list {
+    text-align: left;
+    display: flex;
+    margin-left: 3.125rem;
+  }
+}
+@media (min-width: 48em) {
+  .header-nav-two .nav-list .nav-item {
+    margin-right: 4vw;
+  }
+  .header-nav-two .nav-list .nav-item a {
+    font-size: 1rem;
+    color: #272930;
+  }
+  .header-nav-two .nav-list .nav-item a:hover {
+    color: #ec5241;
+  }
+}
+.header-nav-two .nav-two-container {
+  margin-top: 5.875rem;
+}
+@media (min-width: 48em) {
+  .header-nav-two .nav-two-container {
+    color: #ec5241;
+    margin-top: 0;
+  }
+}
+.header-humberger-menu {
+  margin-left: auto;
+  cursor: pointer;
+  z-index: 10;
+  padding: 0.3125rem;
+  background-color: #fff;
+}
+@media (min-width: 48em) {
+  .header-humberger-menu {
+    display: none;
+  }
+}
+.header-humberger-menu .bar {
+  width: 1.5rem;
+  height: 0.25rem;
+  display: block;
+  background-color: #272930;
+  transition: all 0.2s ease-in-out;
+}
+.header-humberger-menu .bar:not(:first-of-type) {
+  margin-top: 0.25rem;
+}
+.header-humberger-menu.active .bar:nth-child(2) {
+  transform: translateX(18.75rem);
+}
+.header-humberger-menu.active .bar:nth-child(1) {
+  transform: translate(0, 0.5rem) rotate(45deg);
+}
+.header-humberger-menu.active .bar:nth-child(3) {
+  transform: translate(0, -0.5rem) rotate(-45deg);
+}
+.footer-partner {
+  background-color: #414045;
+  padding: 1.5rem;
+}
+@media (min-width: 48em) {
+  .footer-partner {
+    padding: 2rem 0 3.0625rem;
+  }
+}
+.footer-partner-title {
+  text-align: center;
+  color: #d4d4d4;
+  font-size: 1.25rem;
+  margin-bottom: 1.4375rem;
+  width: 34%;
+  margin-inline: auto;
+}
+.footer-partner-title::after {
+  display: block;
+  content: '';
+  background-color: #d4d4d4;
+  width: 1.25rem;
+  height: 0.125rem;
+  margin-inline: auto;
+  margin-top: 0.6875rem;
+}
+.footer-partner-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  row-gap: 0.75rem;
+}
+@media (min-width: 48em) {
+  .footer-partner-container {
+    gap: 4%;
+  }
+}
```

Comment:
- [ ] Kindly organize all your media queries at the bottom of your stylesheet. This is so that you have them all together and not sprinkled across your stylesheet.